en/config/actuators.md

```diff
@@ -1,139 +1,413 @@
+
+The displayed elements depend on the [selected frame](../config/airframe.md), with outputs mapped by default as shown in the [Airframe Reference](../airframes/airframe_reference.md).
+
 :::note
-The *Actuators* view is only displayed if dynamic control allocation is enabled using the [SYS_CTRL_ALLOC](../advanced_config/parameter_reference.md#SYS_CTRL_ALLOC) parameter.
-This replaces geometry and mixer configuration files with configuration using parameters.
-You should also ensure that the appropriate airframe type is selected using [CA_AIRFRAME](../advanced_config/parameter_reference.md#CA_AIRFRAME).
+The *Actuators* view is only displayed if _dynamic control allocation_ is enabled, which replaces geometry and mixer configuration files with parameters.
+This is expected to be enabled by default in PX4 v1.13 (currently disabled in `master`).
 
-The easiest way to try this out in simulation is to use the following make target, which has control allocation pre-enabled:
+To enable this feature, set the parameter [SYS_CTRL_ALLOC=1](../advanced_config/parameter_reference.md#SYS_CTRL_ALLOC) and make sure the correct frame type is set in [CA_AIRFRAME](../advanced_config/parameter_reference.md#CA_AIRFRAME).
+You may need to restart *QGroundControl*. <!-- https://github.com/PX4/PX4-Autopilot/blob/master/src/lib/systemlib/system_params.c#L298 -->
+
+The easiest way to try this out in simulation is to use any of the following gazebo `make` targets, which have control allocation pre-enabled:
 ```
 make px4_sitl gazebo_iris_ctrlalloc
+make px4_sitl gazebo_typhoon_h480_ctrlalloc
 ```
 :::
 
-After selecting an [airframe](../config/airframe.md) you will generally need to configure the specific geometry, assign actuators to outputs, and test the actuator response.
-This can be done in *QGroundControl*, under the **Vehicle Setup > Actuators** tab.
+Open the view in *QGroundControl* here: **"Q" (app menu) > Vehicle Setup > Actuators** (tab).
+
+## Overview
 
-The multicopter configuration screen looks like this.
+The view has three sections:
+- [Geometry](#geometry): Configure the geometry for the [selected airframe](../config/airframe.md).
+  This includes number, position and properties of [motors](#motor-geometry) and also the number and properties of [control surfaces](#control-surfaces-geometry) and [motor tilt servos](#motor-tilt-servo-geometry).
+- [Actuator Outputs](#actuator-outputs): Assign motors, control surfaces, and other actuators to specific output.
+- [Actuator Testing](#actuator-testing): Test that motors and actuators move in the direction / speed as expected.
+
+A quadcopter might have an setup screen similar to the one shown below.
+This defines a 4 motor copter with X-geometry, and maps each of the motors to DShot ESC.
+It also maps PWM400 AUX outputs for controlling a parachute and landing gear as well.
```

Comment:
Nope. You can do "It maps .... as well", or have it as it was. But the addition at the end is redundant.

```suggestion
It also maps PWM400 AUX outputs for controlling a parachute and landing gear.
```